galaxy/management/commands/generate_galaxy_test_data.py

```diff
@@ -0,0 +1,299 @@
+                unix_name=f"galaxy-club-{i}", name=f"club-{i}"
+            )
+
+    def make_users(self):
+        self.users = {}
+        self.picts = {}
+        for i in range(self.USER_ID_MIN, self.USER_ID_MAX):
+            u = User.objects.create_user(
+                username=f"galaxy-user-{i}",
+                email=f"{i}@galaxy.test",
+                first_name="Citizen",
+                last_name=f"n°{i}",
+            )
+            self.users[i] = u
+
+            self.logger.info(f"Registering {u}")
+            Subscription.objects.create(
+                member=u,
+                subscription_start=Subscription.compute_start(
+                    self.now - timedelta(days=self.USER_ID_MAX - i)
+                ),
+                subscription_end=Subscription.compute_end(duration=2),
+            )
+
+            # Create twice as many pictures as users
+            self.picts[i] = Picture.objects.create(
+                owner=self.users[i],
+                name=f"galaxy-picture {u} 1",
+                is_moderated=True,
+                is_folder=False,
+                parent=self.galaxy_album,
+                is_in_sas=True,
+                file=ContentFile(RED_PIXEL_PNG),
+                compressed=ContentFile(RED_PIXEL_PNG),
+                thumbnail=ContentFile(RED_PIXEL_PNG),
+                mime_type="image/png",
+                size=len(RED_PIXEL_PNG),
+            )
+            self.picts[i].file.name = self.picts[i].name
+            self.picts[i].compressed.name = self.picts[i].name
+            self.picts[i].thumbnail.name = self.picts[i].name
+            self.picts[i].save()
+            self.picts[i**2] = Picture(
+                owner=self.users[i],
+                name=f"galaxy-picture {u} 2",
+                is_moderated=True,
+                is_folder=False,
+                parent=self.galaxy_album,
+                is_in_sas=True,
+                file=ContentFile(RED_PIXEL_PNG),
+                compressed=ContentFile(RED_PIXEL_PNG),
+                thumbnail=ContentFile(RED_PIXEL_PNG),
+                mime_type="image/png",
+                size=len(RED_PIXEL_PNG),
+            )
+            self.picts[i**2].file.name = self.picts[i**2].name
+            self.picts[i**2].compressed.name = self.picts[i**2].name
+            self.picts[i**2].thumbnail.name = self.picts[i**2].name
+            self.picts[i**2].save()
+
+            # Make godfathers
+            if i > self.USER_ID_MIN + 200:
+                for j in range(i - 200, i, 14):  # this will loop 14 times
+                    if (i / 10) % 10 == (i + j) % 10:
+                        u1 = self.users[i]
+                        u2 = self.users[j]
+                        self.logger.info(f"Making {u2} the godfather of {u1}")
+                        u1.godfathers.add(u2)
+
+    def make_club_memberships(self):
+        # Make club memberships
+        for i in range(1, 11):  # users can be in up to 20 clubs
```

Comment:
What I see here is that users can be in up to 10 clubs, not 20.

Reply:
Explained in the documentation comment.